vignettes/articles/scripting-tasks-config.Rmd

```diff
@@ -0,0 +1,387 @@
+---
+title: "Scripting task configuration files"
+---
+
+```{r, setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+## Introduction
+
+The `tasks.json` configuration file is a the source of truth for validating a
+hub and is also one of the most complex aspects of a hub. This configuration
+file is a formal representation of an official collaborative forecasting
+challenge that allows direct and unambiguous evaluation and comparison of model
+outputs. It allows hub administrators to set a clear standard for model
+submissions.
+
+While we provide [a hubTemplate](https://github.com/hubverse-org/hubTemplate/)
+for use to get started with setting up a hub, JSON files are easy for a computer
+to read, but not so easy for a human to read and less so to write. The hard part
```

Comment:
The main goal of this suggestion is to simplify sentence structure.  But also, I felt like there was a little bit of a missing connection between using the template hub and the difficulty of manually editing json files.  For example, I might still recommend that someone build a hub starting from the template regardless of whether they plan to edit the tasks.json file manually or use these functions to create it.
```suggestion
JSON files are easy for a computer to read, but not so easy for a human to read and write. The hard part
```